File: app/src/main/java/shaishav/com/bebetter/Data/models/Usage.java
package shaishav.com.bebetter.Data.models;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Shaishav on 26-06-2016.
 */
public class Usage {

    long id,date,usage;

    @SerializedName("_id")
    String server_id;

    public long getId(){
        return id;
    }

    public void setId(long id){
        this.id = id;
    }

    public void setDate(long date){this.date = date;}

    public long getDate(){return date;}

    public void setUsage(long usage){this.usage = usage;}

    public long getUsage(){return usage;}

    public String getServer_id(){return server_id;}

    public void setServer_id(String server_id){this.server_id = server_id;}
}

File: app/src/main/java/shaishav/com/bebetter/data/models/Usage.kt
/*
 * Copyright (c) 2018 Shaishav Gandhi
 *  Licensed under the Apache License, Version 2.0 (the "License");
 *  You may not use this file except in compliance with the License.
 *  You may obtain a copy of the License at
 *
 *  http://www.apache.org/licenses/LICENSE-2.0
 *
 *  Unless required by applicable law or agreed to in writing, software
 *  distributed under the License is distributed on an "AS IS" BASIS,
 *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *  See the License for the specific language governing permissions
 *  and limitations under the License.
 */

package shaishav.com.bebetter.data.models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by shaishav.gandhi on 1/8/18.
 */
@Parcelize data class Usage(val id: Long, val date: Long, val usage: Long): Parcelable {
}